Octokit.Tests/Reactive/ObservableIssuesClientTests.cs

```diff
@@ -329,11 +329,61 @@ public void EnsuresArgumentsNotNull()
             var gitHubClient = Substitute.For<IGitHubClient>();
             var client = new ObservableIssuesClient(gitHubClient);
 
-            Assert.Throws<ArgumentNullException>(() => client.Create(null, "name", new NewIssue("title")));
-            Assert.Throws<ArgumentException>(() => client.Create("", "name", new NewIssue("x")));
-            Assert.Throws<ArgumentNullException>(() => client.Create("owner", null, new NewIssue("x")));
-            Assert.Throws<ArgumentException>(() => client.Create("owner", "", new NewIssue("x")));
-            Assert.Throws<ArgumentNullException>(() => client.Create("owner", "name", null));
+            Assert.Throws<ArgumentNullException>(() => client.Update(null, "name", 42, new IssueUpdate()));
+            Assert.Throws<ArgumentException>(() => client.Update("", "name", 42, new IssueUpdate()));
+            Assert.Throws<ArgumentNullException>(() => client.Update("owner", null, 42, new IssueUpdate()));
+            Assert.Throws<ArgumentException>(() => client.Update("owner", "", 42, new IssueUpdate()));
+            Assert.Throws<ArgumentNullException>(() => client.Update("owner", "name", 42, null));
+        }
```

Comment:
@shiftkey 
Why is this function showing changes.
I had opened a seperate PR to change it which has been merged.
The function has changed also [here](https://github.com/octokit/octokit.net/blob/master/Octokit.Tests/Reactive/ObservableIssuesClientTests.cs#L327)